### PR TITLE
fire segmentoly

### PR DESCRIPTION
**Pull request by Ilya-Krylov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ inference.
     - [Person Re-Identification](pytorch_toolkit/object_reidentification/person_reidentification)
     - [Vehicle Re-Identification](pytorch_toolkit/object_reidentification/vehicle_reidentification)
   * [Human Pose Estimation](pytorch_toolkit/human_pose_estimation)
-  * [Instance Segmentation](pytorch_toolkit/instance_segmentation)
+  * [Instance Segmentation](pytorch_toolkit/instance_segmentation_2)
```
Comment (druzhkov-paul) [inline README.md:38]: Do we need these `_2` suffixes as long as we have only one training engine now?

**Review by druzhkov-paul — COMMENTED** [review]
😢

**Comment by snosov1** [comment]
> 😢

“Many shall be restored that now are fallen, and many shall fall that now are in honor.” HORACE—ARS POETICA.

File: README.md (hunk 1 above)
Comment (Ilya-Krylov) [reply README.md]: removed

**Review by druzhkov-paul — APPROVED** [review]

**Review by DmitriySidnev — APPROVED** [review]

**Review by snosov1 — APPROVED** [review]

**Pull request merged**